lib/src/layer/polyline_layer.dart

```diff
@@ -22,6 +46,12 @@ class Polyline {
   final StrokeJoin strokeJoin;
   final bool useStrokeWidthInMeter;
 
+  /// Custom value that can identify this particular [Polyline] after hit
+  /// detection
+  ///
+  /// See [PolylineLayer.hitNotifier] for more info.
+  final TapKeyType? hitKey;
+
```

Comment:
I see, you also removed the Polyline.onTap for the simplest of use-cases. I'm cool with that, we can always add more stuff in later.

Reply:
To do that, you'd have to add back in a `GestureDetector`, and at that point, where do you stop :D - might as well add a hold and secondary, and then a `MouseRegion`, etc.

Reply:
Feels like my earlier ramblings got to you. Hope you're not getting nightmares :hugs: :hide:

Reply:
Not at all, you were right as usual. Please do keep correcting me, and I'll try to listen sooner 😂